### DISCUSSION TITLE
如何打分词符号

### DISCUSSION THREAD

**Discussion by 54145a** [fcitx5-android/fcitx5-android#557] [Q&A | 问答] [answered]
如题，请问如何打分词符号

**Comment by rocka** [comment, marked as answer]
如果你指的是拼音输入法中分隔音节的符号 `'` ，可以长按或下划 <kbd>z</kbd> 键输入。另外现在输入法内置的拼音可以自动对输入进行分节，一般不需要手动输入分隔符号。
